fiftyone/core/collections.py

```diff
@@ -3345,9 +3496,10 @@ def _build_aggregation(self, aggregations):
     def _process_aggregations(self, aggregations, result, scalar_result):
         results = []
         for idx, agg in enumerate(aggregations):
-            try:
-                results.append(agg.parse_result(result[str(idx)][0]))
-            except:
+            _result = result[str(idx)]
+            if _result:
+                results.append(agg.parse_result(_result[0]))
```

Comment:
@benjaminpkane I didn't want to have a naked `try-except` here because I'd like users to get errors if the run an invalid aggregation. I suspect you added this to deal with edge cases like empty fields/samples that may exist in the App. But, I believe I have explicitly handled those correctly here so that no App errors will arise.

But, your eyes on this would be appreciated.

Reply:
Yes, these changes explicitly handle everything. Thanks!